crates/dwallet-mpc-centralized-party/src/lib.rs

```diff
@@ -68,6 +68,16 @@ pub struct CentralizedDKGWasmResult {
     pub centralized_secret_output: Vec<u8>,
 }
 
+pub fn network_dkg_public_output_to_protocol_pp_inner(
```

Comment:
I'm not sure where this is from, are these @ItayLevyOfficial changes?
Yeah probably I merged his cache PP pr.